gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
```

Comment:
Sorry, forgot to mention this:
Running the task changes this to bin automatically

Reply:
Fixed: a5c7291
Also updated the instructions : )